Add Quarkus

derkoe opened the pull request:
This adds the two sample applications in [Quarkus](https://quarkus.io/) and results running the benchmarks on a ZBook Studio G3 on Fedora 32.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/micronaut-projects/micronaut-comparisons?pullRequest=3) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/micronaut-projects/micronaut-comparisons?pullRequest=3) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/micronaut-projects/micronaut-comparisons?pullRequest=3) it.</sub>

derkoe commented:
Quarkus startup time and time to first response is a bit better. It is a lot slower than the others with the JHipster sample - this seems to be caused by the authentication - I'll try to send them an issue regarding this.

JasonTypesCodes commented:
Thanks @derkoe!

It looks like the Quarkus JHipster application might still have the bcrypt implementation in place (I'm guessing from the contents of [BCryptPasswordHasher](https://github.com/micronaut-projects/micronaut-comparisons/pull/3/files#diff-140e140d2e7dd20e1c5939468f1a56d4)).  We intentionally removed that from the SB and Micronaut examples because it is computationally expensive and negates the usefulness of the comparison.  You will likely get better performance by removing it.

derkoe commented:
I'll try without the bcrypt - thanks for the hint

derkoe commented:
I have changed the Quarkus implementation to use plain passwords and added caching to the user/authorities entities. 

This makes Quarkus a lot faster - but not as fast as Spring or Micronaut.

All JHipster benchmark times have changed because this time I was re-running them with the charger plugged in (last time battery only).
